### PR TITLE
Add examples for entity scripts that set up audio reverb zones

### PR DESCRIPTION
**Pull request by ericrius1**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: f7f7c18d721aa04d838b8b3fdef62769808dd961 available at: https://gist.github.com/dfd567ccc1f5fdc2d7f3a4498566bae7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 048d745af90710f3685798cc91a2c35c9cd8e185 available at: https://gist.github.com/dfd567ccc1f5fdc2d7f3a4498566bae7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.dmg

File: examples/example/audio/largeHall.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+//  Created by Freidrica on 4/1/16.
+//  Copyright 2016 High Fidelity, Inc.
+//
+//  This entity script invokes reverb upon entering an entity acting as a trigger zone
+
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+(function() {
```
Comment (ZappoMan) [inline examples/example/audio/largeHall.js]: indent is wrong on this file... please use 4 space indents

File: examples/example/audio/largeHall.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    lateMixRight: 90,
+    wetDryMix: 90,
+});
+​
+​
+​
+  var panel = new Panel(10, 160);
+​
+var parameters = [
+//    { name: "bandwidth", min: 1000, max: 12000, units: " Hz" },
```
Comment (ZappoMan) [inline examples/example/audio/largeHall.js]: what is this all about? all of the content of this array is commented out...

**Comment by kevinmthomas-carpool** [comment]
Successful smoke test utilizing a Mac on OS X El Capitan.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9cac249122bd0d0889c6b9122a51b7b33cdb33b2 available at: https://gist.github.com/dfd567ccc1f5fdc2d7f3a4498566bae7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 21f6d7bee350542308b73f006251961660d855fc available at: https://gist.github.com/dfd567ccc1f5fdc2d7f3a4498566bae7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2be2f940eabceeb5d3e0eb9ef5353a864786e90e available at: https://gist.github.com/dfd567ccc1f5fdc2d7f3a4498566bae7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0e08b15db7088c04c545cb048aa87eda9673f773 available at: https://gist.github.com/dfd567ccc1f5fdc2d7f3a4498566bae7<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7548.dmg

**Comment by thoys** [comment]
## QA Test results

- large room echo's nicely you can barely hear each other normally :+1: 
- small room has a very small reverb :+1: 

Both zone scripts function great with entering and leaving :+1: .

Tested together with MichelleLeckrone

**Comment by thoys** [comment]
@ericrius1 , next time could you please add testing notes? Not all testers will see that it is JS only :)

**Pull request merged**